flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/plan/stream/sql/SetOperatorsTest.xml

```diff
@@ -40,6 +40,34 @@ GroupAggregate(groupBy=[c], select=[c])
       +- Exchange(distribution=[hash[f]])
          +- Calc(select=[f])
             +- TableSourceScan(table=[[default_catalog, default_database, T2, source: [TestTableSource(d, e, f)]]], fields=[d, e, f])
+]]>
+    </Resource>
+  </TestCase>
+  <TestCase name="testIntersectAll">
```

Comment:
this case should be in second commit ?

Reply:
Maybe I should just squash second and third to a one commit.